src/core/PhysicsDetector.ttslua

```diff
@@ -0,0 +1,7 @@
+-- will notify the user to enable physics if it appears to not be fully enabled
+
+-- this event should only fire if physics aren't fully enabled
+function onCollisionExit()
+  broadcastToAll("Physics disabled?", "Orange")
```

Comment:
I'd add a little bit to the broadcast like "(Check chat log)" or something just to make sure people see the longer message

Reply:
Fair point, will add that